src/main/java/com/redhat/cloud/notifications/db/ApplicationResources.java

```diff
@@ -154,10 +158,12 @@
     }
 
     public Uni<Boolean> deleteEventTypeById(UUID eventTypeId) {
-        String query = "DELETE FROM public.event_type WHERE id = $1";
 
-        return runDeleteQuery(eventTypeId, query);
+        // first remove endpoints for the eventType, then the type itself
 
+        String ep_query = "DELETE FROM endpoint_targets WHERE event_type_id = $1";
```

Comment:
You won't need to do this anymore after: https://github.com/RedHatInsights/notifications-backend/pull/110/files#diff-cfb416495b160128789c37c83b6a95cd34abc5818859735bcb5354492b9fe51cR20-R24